linters/remark-lint/test_data/.remarkrc.yaml

```diff
@@ -0,0 +1,5 @@
+plugins:
```

Comment:
lift this up one level to auto-recommend to people

Reply:
done